src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.VSCodeWorkspaces/VSCodeHelper/VSCodeInstances.cs

```diff
@@ -74,7 +74,7 @@ public static void LoadVSCodeInstances()
                 if (Directory.Exists(path))
                 {
                     var files = Directory.GetFiles(path)
-                        .Where(x => (x.Contains("code", StringComparison.OrdinalIgnoreCase) || x.Contains("VSCodium", StringComparison.OrdinalIgnoreCase))
+                        .Where(x => (x.Contains("code", StringComparison.OrdinalIgnoreCase) || x.Contains("codium", StringComparison.OrdinalIgnoreCase))
```

Comment:
Why remove "VSCodium"? Isn't that still used by some versions?

Reply:
The question here is if this won't break support for older versions that use "codium" instead of "VSCodium". I don't think there's a reason to remove that support.

Reply:
@eternalphane , mentioning you just in case the notification didn't reach you.

Reply:
It's IgnoreCase search. It is widening the search, so should be ok

Reply:
Misread these lines. Sorry.